documentation/src/main/asciidoc/topics/proc_registering_sci_remote_caches.adoc

```diff
@@ -0,0 +1,44 @@
+[id='registering-sci-remote-caches_{context}']
+= Registering serialization context initializers with {brandname} Server
+
+You must register your Protobuf schema with {brandname} Server before you can marshall user types to and from remote caches.
```

Comment:
That's only true if you wish todo anything on the server side that requires knowledge of the user types protobuf format. If no transformations, i.e. transcoding or marshalling, occur then registration can be skipped as the values are just handled as  a simple `byte[]`.

Reply:
As this is about SerializationContextinitializer it is protobuf so the schema is needed